auto_route/lib/src/router/controller/auto_router_delegate.dart

```diff
@@ -69,9 +71,7 @@ class AutoRouterDelegate extends RouterDelegate<UrlState> with ChangeNotifier {
 
   /// Forces a url update
   static reportUrlChanged(BuildContext context, String url) {
-    Router.of(context)
-        .routeInformationProvider
-        ?.routerReportsNewRouteInformation(
+    Router.of(context).routeInformationProvider?.routerReportsNewRouteInformation(
```

Comment:
There are a few formatting changes in this PR due to the configured 120 line length. 
I think this file was not formatted with the configured 120 lines setting.